packages/server/src/http/internals/HTTPHandlerOptions.ts

```diff
@@ -49,3 +57,13 @@ export type HTTPHandlerOptions<
          **/
         createContext: CreateContextFn<TRouter, TRequest, TResponse>;
       });
+
+export interface ResolveHTTPRequestOptions<
+  TRouter extends AnyRouter,
+  TRequest extends HTTPRequest,
+> extends HTTPHandlerOptionsBase<TRouter, TRequest> {
+  createContext: () => Promise<inferRouterContext<TRouter>>;
```

Comment:
🤔 doesn't this mean that `createContext` can't receive any fun input? Perhaps it's possible to create a partial of a function and "load" stuff into it before sending it to `resolveHttpResponse` somehow?

Reply:
No, it just means that the wrapper fn is responsible of arguments. Check `requestHandler` for reference